sandbox/apparmor/notify/version.go

```diff
@@ -42,8 +45,34 @@ var (
 	// the next version in the list.
 	versionLikelySupportedChecks = map[ProtocolVersion]func() bool{
 		3: func() bool {
```

Comment:
a bit surprised but coverage seems to indicated that the functions in this map are not tested